server/routes/bookers/booker/bookerPrisonerDetailsController.ts

```diff
@@ -11,7 +11,7 @@ type Visitor = {
   restrictions: ContactDto['restrictions']
   active: boolean
 }
-export default class BookerDetailsController {
+export default class BookerPrisonerDetailsController {
```

Comment:
This now has the prisoner/visitor management bits of the previous `BookerDetailsController`